### ISSUE TITLE
Separate Featured Artists articles from the rest of news articles on homepage

### ISSUE THREAD

**Issue by WalterToro** [ppy/osu-web#7685]
As it currently stands, a news article has a maximum of 8 days to be on the top 3 news articles on the homepage (assuming only FA articles and nothing else), this reduces spotlight time for official contests / world cups articles / community posts / etc.
With the FA related news articles being more than half of the current 2021 news posts as seen on this table:

| Month | FA posts | Total |
| :-: | :-: | :-: |
| January | 9 | 13 |
| February | 8 | 13 |
| March | 9 |16 |
| April | 8 | 18 |
| May | 9 | 19 |

I think it should be better to create another section on the homepage just for FA related stuff, you can fill this with FA news articles and a random selection of FA beatmaps to spotlight.

**Comment by RockRoller01** [comment]
Posting updated numbers on this as I feel like this issue has gotten worse over time. There is basically a newspost every 2 days, however sadly it doesn't work out like that in the real world and often they clump up on e.g. the end of the week, where it would be nice to have that one more unblocked day. Not to mention I personally think this is a good improvement for the homepage/news system.

|  month  | FA  | other | total |
| :-----: | :-: | :---: | :---: |
| jun '21 |  9  |   5   |  14   |
| jul '21 |  9  |   8   |  17   |
| aug '21 |  8  |   6   |  14   |
| sep '21 |  8  |   5   |  13   |
| oct '21 |  9  |   4   |  13   |
| nov '21 |  8  |   5   |  13   |
| dez '21 |  8  |   8   |  16   |
| jan '22 |  8  |   5   |  13   |
| feb '22 |  8  |   7   |  15   |
| mar '22 |  9  |   4   |  13   |
| apr '22 |  9  |   6   |  15   |
| may '22 |  8  |   6   |  14   |
| jun '22 |  9  |   2   |  11   |
| jul '22 |  8  |   8   |  16   |
| aug '22 |  9  |   8   |  17   |
| sep '22 |  8  |  10   |  18   |

**Comment by pishifat** [comment]
this can prob be closed since fa news pace is slower

**Comment by cl8n** [comment]
^ doesn't seem so bad anymore. #9614 is probably a better issue to keep open if the goal is to make sure that ongoing events stay pinned somewhere
